src/StreamJsonRpc.Shared/Reflection/MethodNameAttribute.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using Microsoft;
+
+namespace StreamJsonRpc
+{
+    /// <summary>
+    /// Attribute which allows a method to have an alternate name.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method, AllowMultiple = false, Inherited = false)]
```

Comment:
Given that `AllowMultiple=false` why is the string resource you added necessary that is prepared to report on multiple of these attributes being applied?